habitat_baselines/rl/models/rnn_state_encoder.py

```diff
@@ -0,0 +1,116 @@
+import torch
+import torch.nn as nn
+
+
+class RNNStateEncoder(nn.Module):
+    def __init__(self, input_size, hidden_size, num_layers=1, rnn_type="GRU"):
```

Comment:
Maybe we can add doc string here and mention what are options for `rnn_type`  and add typing.